pkgs/applications/networking/remote/vmware-horizon-client/default.nix

```diff
@@ -0,0 +1,117 @@
+{ stdenv, buildFHSUserEnv, fetchurl, makeDesktopItem, libxslt, atk
+, fontconfig, freetype, gdk_pixbuf, glib, gtk2, libudev0-shim, libxml2
+, pango, pixman, libX11, libXext, libXinerama, libXrandr , libXrender
+, libXtst, libXcursor, libXi, libxkbfile , libXScrnSaver, zlib, liberation_ttf
+}:
+
+let
+    version = "4.10.0-11053294";
+
+    vmwareBundle64 = fetchurl {
+        url = https://download3.vmware.com/software/view/viewclients/CART19FQ4/VMware-Horizon-Client-4.10.0-11053294.x64.bundle;
+        sha256 = "21617d8223aad7d282967b07dbf09ec5d5dd389d706063c0b791b20bdfe425bb";
+    };
+
+    vmwareBundleUnpacker = fetchurl {
+        url = https://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/eclass/vmware-bundle.eclass;
```

Comment:
Is this url guaranteed to be stable when gentoo updates this file ?

Reply:
The URL has been updated to be stable.